detection-rules/attachment_office_remote_doc_template.yml

```
@@ -0,0 +1,24 @@
+name: "Attachment: Office document loads remote document template"
+description: |
+  Recursively scans archives and Office documents to detect remote document template injection.
+references:
+  - "https://delivr.to/payloads?id=c7a7195e-0de3-428d-a32c-5fd59a3012da"
+type: "rule"
+severity: "high"
```

```suggestion
severity: "medium"
```
since we don't know for sure this is bad, medium might be better